.pre-commit-config.yaml

```diff
@@ -0,0 +1,55 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+      - id: end-of-file-fixer
+        exclude: ^internal/(?:go|pprof)/
+      - id: trailing-whitespace
+        exclude: ^internal/(?:go|pprof)/
+  - repo: https://github.com/Lucas-C/pre-commit-hooks
+    rev: v1.3.0
+    hooks:
+      - id: insert-license
+        name: Insert license in comments (Go)
+        args:
+          - --license-filepath=scripts/LICENSE.txt
+          - --comment-style=//
+          - --fuzzy-match-generates-todo
+        types: [go]
+        exclude: ^internal/(?:go|pprof)/
+      - id: insert-license
+        name: Insert license in comments (Shell)
+        args:
+          - --license-filepath=scripts/LICENSE.txt
+          - --comment-style=#
+          - --fuzzy-match-generates-todo
+        types: [shell]
+        exclude: ^internal/(?:go|pprof)/
+  - repo: https://github.com/golangci/golangci-lint
+    rev: v1.48.0
```

Comment:
Is there a way to use renovate to keep track of these versions as well? We will have version drifts otherwise.

Reply:
Yes! Remember parca-dev/.github#5?

Reply:
Ah yeah, true! 👍